components/Themes/src/MDCFeatureHighlightColorThemer.m

```diff
@@ -0,0 +1,27 @@
+/*
+ Copyright 2015-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCFeatureHighlightColorThemer.h"
+
+@implementation MDCFeatureHighlightColorThemer
+
++ (void)applyTheme:(MDCTheme *)theme
+    toFeatureHighlight:(MDCFeatureHighlightViewController *)featureHighlightVC {
```

Comment:
Objective-c typically isn't big on abbreviations, please use Controller instead of VC.

Please have the method implementation match the declaration.  Currently we have featureHighlight in the header and featureHighlightVC here.